src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -123,6 +123,15 @@ std::vector<ActorID> GcsActorScheduler::CancelOnNode(const ClientID &node_id) {
   return actor_ids;
 }
 
+void GcsActorScheduler::CancelOnLeasing(const ClientID &node_id,
+                                        const ActorID &actor_id) {
+  /// Note(sang): It is currently effective only inside RequestWorkerLease callback.
```

Comment:
What does this comment mean?

Reply:
Sorry I realized the comment is really bad. The comment is added based on this feedback https://github.com/ray-project/ray/pull/9215/files#r449460230. 

I will modify to
```
// NOTE(sang): This method does not currently cancel the outstanding lease request.  
It only removes leasing information from the internal state so that RequestWorkerLease ignores the response from raylet. 
```

This comment is added to avoid the situation that people use this method thinking it will cancel the lease request which is actually not true for now